### PR TITLE
do not merge - just testing something

### PR DESCRIPTION
**Pull request by pieh**
this is just for testing purposes - I will close this later

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -50,6 +50,7 @@ aliases:
   test_template: &test_template
     steps:
       - checkout
+      - run: node -v
```
Comment (DSchau) [inline .circleci/config.yml]: 😬 hopefully the executors are working like we'd expect! Good idea.

**Comment by pieh** [comment]
This is just to test something that came up in https://github.com/gatsbyjs/gatsby/pull/12719 - because according to info there - it should fail for node6 (as some module is node8+ only), but it passed there :S

**Comment by pieh** [comment]
It reports expected node version - so not sure why test in PR linked above doesn't fail in some ways if it needs node 8+ stuff - hmm, let's check something more

**Comment by DSchau** [comment]
@pieh that package could be following our approach, haha. e.g. "soft deprecating" but still supporting?

**Comment by pieh** [comment]
That's also possible. Maybe just parts of the package that we use might not require 8+?

**Comment by pieh** [comment]
Ok, so tests that is designed to fail in node 6 and pass in node 8+ works - so our tests doesn't seem to be wonky after all

**Pull request closed without merging**